docs/source/io_formats/data_wmp.rst

```diff
@@ -0,0 +1,102 @@
+.. _io_data_wmp:
+
+=================================
+Windowed Multipole Library Format
+=================================
+
+**/version** (*char[]*)
+  The format version of the file.  The current version is "v0.2"
+
+**/nuclide/**
+    - **broaden_poly** (*int[]*)
+        If 1, Doppler broaden curve fit for window with corresponding index.
+        If 0, do not.
+    - **curvefit** (*double[][][]*)
+        Curve fit coefficients. Indexed by (reaction type, coefficient index,
+        window index).
+    - **data** (*complex[][]*)
+        Complex poles and residues. Each pole has a corresponding set of
+        residues. For example, the :math:`i`-th pole and corresponding residues
+        are stored as
+        
+        .. math::
+        \text{data}[:,i] = [\text{pole},~\text{residue}_1,~\text{residue}_2,
+        ~\ldots]`
```

Comment:
You need to separate this by a blank line and indent the equation. Also, there is a backtick at the end that looks like it should be removed.